### DISCUSSION TITLE
How to create a distribution tarball?

### DISCUSSION THREAD

**Discussion by salmankhilji** [kovidgoyal/kitty#3649] [Q&A] [answered]
After compiling kitty from source manually, how do I create a my own private release tarball so that my colleagues do not have to compile the same from source?

(Alternately, I can wait for the next release that have bug fixes I am relying on.)

I am not a Python expert.  I found `run_sdist` within `publish.py` but this routine is not called from any place.

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/build.html#note-for-linux-macos-packagers
